modules/7.2.4/module.yaml

```diff
@@ -16,6 +16,12 @@ artifacts:
     - name: jboss-eap-7.2.4-patch.zip
       path: jboss-eap-7.2.4-patch.zip
       md5: ebdc5f99141d5e68c42116cf1fef6e60
+    - name: jbeap-17583.zip
```

Comment:
these should all be like: 

- name: jbeap-17583
  target: jbeap-17583.zip
  md5: ebdc5f99141d5e68c42116cf1fef6e60

Then you'll need to cekit-cache add them before you can build.